diff --git a/CMakeLists.txt b/CMakeLists.txt
index fd3d39a..610a2c6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -170,5 +170,5 @@ add_subdirectory(docs)
 # Install cmake search helper for this library
 ########################################################################
 install(FILES cmake/Modules/dabConfig.cmake
-    DESTINATION lib/cmake/grdab
+    DESTINATION lib${LIB_SUFFIX}/cmake/grdab
 )
